playbooks/roles/libvirt_setup/tasks/main.yml

```diff
@@ -22,6 +22,10 @@
 - name: 'restart libvirt'
   service: name=libvirtd state=restarted
 
+- name: 'install libvirt-python dependency'
+  yum:
+    name: libvirt-python
+
 - name: 'create the provisioning libvirt network'
```

Comment:
When running outside of vagrant box, I got error on Centos 7.2:
"The lxml module is not importable. Check the requirements."

The following helped:
```yaml
- name : 'install python-lxml'
  yum:
    name: python-lxml
```